parse/PythonParser.cpp

```diff
@@ -40,6 +40,8 @@ namespace {
     void translate(import_error const& e) {
         PyErr_SetString(PyExc_ImportError, e.what());
     }
+
+    static const bool STATIC_FALSE = false;
```

Comment:
```suggestion
    constexpr bool STATIC_FALSE = false;
```

Reply:
Ok, but I need to test if reference to constexpr have enought lifetime.

Reply:
If not, there is something wrong with the compiler...

Reply:
Yes, it helps too.